pkg/provider/aws/static_unit_test.go

```diff
@@ -77,19 +76,7 @@ func Test_GenerateStaticUnitResources(t *testing.T) {
 				return
 			}
 
-			tiedResources, found := aws.GetResourcesDirectlyTiedToConstruct(unit)
```

Comment:
part of refactor?

Reply:
Yeah, because some of the signatures on the graph were changed, I had to change the test. Instead of updating the functions on the dag, I just switched it to use the ResourcesExpectation assert method instead.